ruby_on_rails/advanced_forms_and_activerecord/installing_psql.md

```diff
@@ -0,0 +1,342 @@
+### Introduction
+
+One of the great things about Rails is that it comes with some sensible defaults so that you can get started quickly. However, SQLite is typically not used for production. One major issue is that many hosting providers have ephemeral file systems which means your database can be deleted.
+
+For the apps written so far, it's probably not a big deal. As we progress to writing larger applications, though, it will become increasingly sensible to make sure that data persists even after a restart or hibernation.
+
+It is also sensible to make sure the application development environment is as close to the production environment as possible. This helps to avoid problems that stem from using different tools in different environments.
+
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+-  What an environment is and how it can impact our application
+-  What environment variables are and how to utilize them
+-  How to install PostgreSQL and set it up
+-  How to set up a bare Rails application to use a specific database
```

Comment:
I really want those full stops 😆 

```suggestion
-  What an environment is and how it can impact our application.
-  What environment variables are and how to utilize them.
-  How to install PostgreSQL and set it up.
-  How to set up a bare Rails application to use a specific database.
```